src/stactools/modis/fragments/MOD10A2/061/raster-bands.json

```diff
@@ -0,0 +1,12 @@
+{
+    "Maximum_Snow_Extent": {
+        "data_type": "uint8",
+        "unit": null,
+        "scale": null
+    },
+    "Eight_Day _Snow_Cover": {
```

Comment:
```suggestion
    "Eight_Day_Snow_Cover": {
```

Reply:
Removed space in name for "raster-bands.json" in MCD64A1, MOD10A1, MOD10A2, MOD13A1, MOD13Q1, MYD10A1, MYD10A2, MYD13A1, and MYD13Q1.